### ISSUE TITLE
astc encoder uses a upper case windows path causing mingw problems

### ISSUE THREAD

**Issue by fire** [ARM-software/astc-encoder#389]
```c++
/* ============================================================================
   Platform code for Visual Studio
============================================================================ */
#if !defined(__clang__) && defined(_MSC_VER)
#define WIN32_LEAN_AND_MEAN
#include <Windows.h>
```

should be

```c++
/* ============================================================================
   Platform code for Visual Studio
============================================================================ */
#if !defined(__clang__) && defined(_MSC_VER)
#define WIN32_LEAN_AND_MEAN
#include <windows.h>
```

**Comment by solidpixel** [comment]
Thanks for the bug report. 

Fixed in f758d218fe91970f28b8a1416a2ec52810ac5fdf

**Comment by solidpixel** [comment]
Backport to 3.x branch: c2aca7d43dbef36de3be961ed3aa9c8abab57ec3
